Makefile

```diff
@@ -52,22 +55,32 @@ test:
 build: clean
 	@echo Building
 	$(GO) build -o dist/mattermod ./cmd/mattermost-mattermod
+	$(GO) build -o dist/jobserver ./cmd/jobserver
```

Comment:
In order to reduce build time (specially on release) may be we can separate these into three targets:

```
build: build-mattermod build-jobserver

build-mattermod:
    $(GO) build ....

build-jobserver:
    $(GO) build ....
```